assets/src/js/admin/reports/components/donor-item/index.js

```diff
@@ -1,44 +1,48 @@
-import PropTypes from 'prop-types'
-import './style.scss'
-import { getBGColor, getInitials } from './utils'
+import PropTypes from 'prop-types';
+import './style.scss';
+import { getBGColor, getInitials } from './utils';
 
-const DonorItem = ({image, name, email, count, total}) => {
-
-    const profile = image ? <img src={image} /> : <div className='donor-initials' style={{backgroundColor: getBGColor()}}>{getInitials(name)}</div>
-    return (
-        <div className='donor-item'>
-            {profile}
-            <div className='donor-info'>
-                <p><strong>{name}</strong></p>
-                <p>{email}</p>
-            </div>
-            <div className='donor-totals'>
-                <p>{count}</p>
-                <p>{total}</p>
-            </div>
-        </div>
-    )
-}
+const DonorItem = ( { image, name, email, count, total, url } ) => {
+	const profile = image ? <img src={ image } /> : <div className="donor-initials" style={ { backgroundColor: getBGColor() } }>{ getInitials( name ) }</div>;
+	return (
+		<a className="donor-link" href={ url }>
+			<div className="donor-item">
+				{ profile }
+				<div className="donor-info">
```

Comment:
@henryholtgeerts Can we add a prefix to each class name `give-`

Reply:
Prefixes added, ref: https://github.com/impress-org/givewp/pull/4429/commits/3615902dd65aaf883772f5031aed91c08c6a314e